hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/InsertIntoHoodieTableCommand.scala

```diff
@@ -51,16 +52,20 @@ case class InsertIntoHoodieTableCommand(logicalRelation: LogicalRelation,
                                         query: LogicalPlan,
                                         partitionSpec: Map[String, Option[String]],
                                         overwrite: Boolean)
-  extends HoodieLeafRunnableCommand {
-  override def innerChildren: Seq[QueryPlan[_]] = Seq(query)
+  extends DataWritingCommand {
```

Comment:
innserChildren can not rendering spark sql global dag
before:
<img width="835" alt="image" src="https://user-images.githubusercontent.com/20125927/226170774-4b708ed5-8065-46c5-b975-7ee2fe47e37a.png">

after:
<img width="580" alt="image" src="https://user-images.githubusercontent.com/20125927/226170782-548425f1-32c1-446b-8710-c169e3ede6f9.png">